### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Thomas-Boi**
Contain:
- Remove references about contributors having to optimize the SVGs manually
- Update the requesting icon section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ First of all, thanks for taking the time to contribute! This project can only gr
 
 <hr>
 <h2 id='requestingIcon'>Requesting an Icon</h2>
-<p>To request an icon, you can create an issue in the repository. Please follow these simple guidelines:</p>
+<p>To request an icon, please create an issue in the repository and follow these guidelines:</p>
```
Comment (Panquesito7) [inline CONTRIBUTING.md:56]:
```suggestion
<p>To request an icon, please create an issue in the repository, and follow these guidelines:</p>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,12 +53,12 @@ First of all, thanks for taking the time to contribute! This project can only gr
...
 <ul>
   <li>Search for other issues already requesting the icon</li>
   <li>If an issue doesn't exist, create an issue naming it "Icon request: <i>name-of-the-icon</i>". </li>
   <li>Please create separate issues for each icon</li>
-  <li><b>Optional:</b> include links where the icon can be found</li>
```
Comment (Panquesito7) [inline CONTRIBUTING.md:61]: I don't think this should be removed. We should know from which sources they took the icon (they could be using a similar or fake icon to the original).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ First of all, thanks for taking the time to contribute! This project can only gr
   <li><i>Optional</i>: Reference the issues regarding the new icon. </li>
   <li>Some bots will check your SVGs. If there are any errors, please fix them as instructed.</li>
   <li>Wait for a maintainer to review your changes. They will run the <a href='#peekBot'><code>peek-bot</code></a> to check your icons.</li>
-  <li>If there are no issues, they will run the <a href='#optimizeBot'><code>optimize-bot</code></a> on your SVGs then merge it using <a href="https://github.com/devicons/devicon/discussions/470">squash merging</a>. If there are any problems, they will let you know and give you a chance to fix it.</li>
+  <li>If there are no issues, they will merge it using <a href="https://github.com/devicons/devicon/discussions/470">squash merging</a>. If there are any problems, they will let you know and give you a chance to fix it.</li>
```
Comment (Panquesito7) [inline CONTRIBUTING.md:76]:
```suggestion
  <li>If there are no issues, they will merge it using <a href="https://github.com/devicons/devicon/discussions/470"><b>squash merging</b></a>. If there are any problems, they will let you know, and give you a chance to fix them.</li>
```

**Review by Panquesito7 — COMMENTED** [review]
👍 Good work, Thomas.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Thomas-Boi) [reply CONTRIBUTING.md:61]: That sounds good 👍. I've added the line again.

This is actually listed in the PR template but for some reason, it's not showing for people to use 🤔

**Comment by Thomas-Boi** [comment]
# IMPORTANT

The latest commit also fixes #693, which renames the `vueStorefront` folder to `vuestorefront`. Git Bash somehow doesn't register the name change as a commitable change => I had to batch it in with other changes. Let's see if it works. If it doesn't, I'll figure out another way

**Review by amacado — APPROVED** [review]

**Comment by amacado** [comment]
@Thomas-Boi i assume the vuestorefront fix was not successful. I approve this request anyway to update the CONTRIBUTING.md.

**Pull request merged**